lib/services_cli.rb

```diff
@@ -15,7 +15,25 @@ def bin
 
     # Path to launchctl binary.
     def launchctl
-      which("launchctl")
+      @launchctl ||= which("launchctl")
+    end
+
+    def launchctl?
+      launchctl.present?
+    end
+
+    # Path to launchctl binary.
```

Comment:
No it's not 😁.

I wonder if we could split these into `launchctl_cli` and `systemctl_cli` classes?